packages/lesswrong/server/authenticationMiddlewares.ts

```diff
@@ -68,21 +69,23 @@ async function mergeAccount(idPath: string, user: DbUser, profile: Profile) {
  * Given the provider-appropriate ways to get user info from a profile, create
  * a function that handles successful logins from that provider
  */
-function createOAuthUserHandler<P extends Profile>(idPath: string, getIdFromProfile: IdFromProfile<P>, getUserDataFromProfile: UserDataFromProfile<P>) {
+function createOAuthUserHandler<P extends Profile>(profilePath: string, getIdFromProfile: IdFromProfile<P>, getUserDataFromProfile: UserDataFromProfile<P>) {
```

Comment:
Did I work on this with you or something? I remember this.

Reply:
I think I remember having worked on a fix for this together, but I think that fix wound up on a lost branch or something because when I came back to the issue I couldn't find it.

Reply:
Lol